### ISSUE TITLE
Progressbar on download

### ISSUE THREAD

**Issue by gutierri** [mfichman/winbrew#13]
This is not really a feature, but it seems to be someone interesting. We can handle errors, and trigger an exception if there are errors.

**Comment by mfichman** [comment]
Yes, I agree. The UI for progress is non-existent at this point. There are a few Python libraries that are good for making text UI progress bars. I will look into that when I have time, but feel free to contribute code if you have something in mind!
